pkg/validation/internal/operatorhub.go

```diff
@@ -74,16 +74,19 @@ func validateBundleOperatorHub(bundle *manifests.Bundle) errors.ManifestResult {
 		return result
 	}
 
-	errs := validateHubCSVSpec(*bundle.CSV)
+	errs, warns := validateHubCSVSpec(*bundle.CSV)
 	for _, err := range errs {
 		result.Add(errors.ErrInvalidCSV(err.Error(), bundle.CSV.GetName()))
 	}
-
+	for _, warn := range warns {
+		result.Add(errors.WarnInvalidCSV(warn.Error(), bundle.CSV.GetName()))
+	}
 	return result
 }
 
-func validateHubCSVSpec(csv v1alpha1.ClusterServiceVersion) []error {
+func validateHubCSVSpec(csv v1alpha1.ClusterServiceVersion) ([]error, []error) {
 	var errs []error
+	var warns []error
```

Comment:
This seems like the wrong type to use? We're not overloading any of the error methods here at all (or implementing anything special on top), and we're ultimately just calling warn.Error() to get the string back out. Seems like this could just be a slice of strings?

Reply:
@kevinrizza it looks like he was just reusing what was already there. The logic assumes a slice of errors. But what you suggest is more correct because the strings are passed into specific `Error`s. +1 to using `[]string` but will that mess anything else up?